src/interpreter.cr

```diff
@@ -0,0 +1,95 @@
+
+struct Interpreter
+  @@shell : String | Nil
+
+  protected def self.shell : String
+    {% if flag?(:darwin) %}
+      @@shell ||= begin
+        Process.find_executable(ENV.fetch("RUBYSHELL", "dash")).as(String)
```

Comment:
Just honouring `RUBYSHELL` for backwards-compatibility with Ruby `ops`?

Reply:
Yeah pretty much. Though, I can remove that if its support was to end with `crops`. Might even be worth establishing an `OPS_SHELL` var or something.